### PR TITLE
feat: add Blockstore minio plugin

### PR DESCRIPTION
**Pull request by brian-smith-tcril**
Adds experimental plugin for minio based blockstore configuration

### CODE REVIEW

File: plugins/tutor-contrib-blockstore-minio/README.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+blockstore `minio <https://min.io/>`__ configuration plugin for `Tutor <https://docs.tutor.overhang.io>`__
+===================================================================================
+
+Installation
+------------
+
+::
+
+    pip install git+https://github.com/brian-smith-tcril/tutor-contrib-blockstore-minio
```
Comment (arbrandes) [inline plugins/tutor-contrib-blockstore-minio/README.rst:9]:
```suggestion
    pip install git+https://github.com/openedx/openedx-tutor-plugins#subdirectory=plugins/tutor-contrib-blockstore-minio
```

File: plugins/tutor-contrib-blockstore-minio/README.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Usage
+-----
+
+::
+
+    tutor plugins enable blockstore_config_minio
```
Comment (arbrandes) [inline plugins/tutor-contrib-blockstore-minio/README.rst:16]:
```suggestion
    tutor plugins enable blockstore-minio
```

File: plugins/tutor-contrib-blockstore-minio/setup.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    author="Brian Smith",
+    description="blockstore minio configuration plugin for Tutor",
+    long_description=load_readme(),
+    packages=find_packages(exclude=["tests*"]),
+    include_package_data=True,
+    python_requires=">=3.7",
+    install_requires=["tutor"],
+    entry_points={
+        "tutor.plugin.v1": [
+            "blockstore_config_minio = blockstore_config_minio.plugin"
```
Comment (arbrandes) [inline plugins/tutor-contrib-blockstore-minio/setup.py:45]:
```suggestion
            "blockstore-minio = blockstore_config_minio.plugin"
```

Same reasoning as the filesystem one.

File: plugins/tutor-contrib-blockstore-minio/setup.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ABOUT = load_about()
+
+
+setup(
+    name="tutor-contrib-blockstore-minio",
+    version=ABOUT["__version__"],
+    url="https://github.com/brian-smith-tcril/tutor-contrib-blockstore-minio",
+    project_urls={
+        "Code": "https://github.com/brian-smith-tcril/tutor-contrib-blockstore-minio",
+        "Issue tracker": "https://github.com/brian-smith-tcril/tutor-contrib-blockstore-minio/issues",
```
Comment (arbrandes) [inline plugins/tutor-contrib-blockstore-minio/setup.py:33]:
```suggestion
    url="https://github.com/openedx/openedx-tutor-plugins",
    project_urls={
        "Code": "https://github.com/openedx/openedx-tutor-plugins",
        "Issue tracker": "https://github.com/openedx/openedx-tutor-plugins/issues",
```

**Review by arbrandes — CHANGES_REQUESTED** [review]

**Comment by brian-smith-tcril** [comment]
@arbrandes all comments addressed by https://github.com/openedx/openedx-tutor-plugins/compare/3f61670d3e670698118942ecdfa15f477e59f319..5bc274b641ee38ed87655fe2329264cbe31f8945

File: plugins/tutor-contrib-blockstore-minio/openedx-common-settings [hunk 5]
```diff
@@ -0,0 +1 @@
+BUNDLE_ASSET_STORAGE_SETTINGS = dict(STORAGE_CLASS="django.core.files.storage.FileSystemStorage", STORAGE_KWARGS=dict(location="/openedx/media/blockstore/", base_url="http://localhost:8000/media/blockstore/"))
```
Comment (arbrandes) [inline plugins/tutor-contrib-blockstore-minio/openedx-common-settings:1]: Same as with the filesystem version: this should either be moved to `patches/` or removed.

**Review by arbrandes — CHANGES_REQUESTED** [review]

File: plugins/tutor-contrib-blockstore-minio/openedx-common-settings (hunk 5 above)
Comment (brian-smith-tcril) [reply plugins/tutor-contrib-blockstore-minio/openedx-common-settings]: https://github.com/openedx/openedx-tutor-plugins/compare/5bc274b641ee38ed87655fe2329264cbe31f8945..e8d8951373df730585ba835127cd5b1f253250f2

File: README.rst [hunk 6]
```diff
@@ -20,6 +20,8 @@ Plugin                               Status (*Experimental*, *Production*, or *D
 ===================================  ======================================================
 tutor-contrib-learner-dashboard-mfe  Experimental
 ===================================  ======================================================
+tutor-contrib-blockstore-minio       Experimental
```
Comment (arbrandes) [inline README.rst:23]: Missed this in the last one: the rows in the table shouldn't have separators between them.

```suggestion
tutor-contrib-blockstore-minio       Experimental
```

File: README.rst [hunk 7]
```diff
@@ -19,9 +19,8 @@ community:
 Plugin                               Status (*Experimental*, *Production*, or *Deprecated*)
 ===================================  ======================================================
 tutor-contrib-learner-dashboard-mfe  Experimental
-===================================  ======================================================
 tutor-contrib-blockstore-filesystem  Experimental
-===================================  ======================================================
+tutor-contrib-blockstore-minio       Experimental
```
Comment (arbrandes) [inline README.rst:23]: ...but the tailing separator is still needed. :)

Comment (arbrandes) [reply README.rst:23]: (Yeah, RST, I know.)

**Review by arbrandes — APPROVED** [review]
Thanks, looks great!

File: README.rst (hunk 6 above)
Comment (brian-smith-tcril) [reply README.rst]: working now https://github.com/openedx/openedx-tutor-plugins/blob/3c72af99c4a955eabe4e48e31710132dba5eb6a1/README.rst

File: README.rst (hunk 7 above)
Comment (brian-smith-tcril) [reply README.rst:23]: I hadn't installed the rst preview vscode extension on this machine since switching all my work over. Now I've got it again

**Pull request merged**